CourseApp/ClassTaskFolder/FileCreator.cs

```diff
@@ -0,0 +1,180 @@
+namespace CourseApp
+{
+    using System;
+    using static System.Math;
+
+    public class FileCreator : Document
+    {
+        private string _extension;
+        private double _weight;
+        private string _weightModificator;
+
+        public FileCreator(string name, string extension, double weight, string weightModificator)
+        : base(name)
+        {
+            Extension = extension;
+            Weight = weight;
+            WeightModificator = weightModificator;
+        }
+
+        public FileCreator()
+        : base()
+        {
+            RandomExtension();
+            RandomWeight();
+            RandomWeightModificator();
```

Comment:
Can we move random values generators outside this class? It's bad when the class does a lot of things.

Reply:
Fixed